opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -966,6 +970,1076 @@ def parse_pptx(file_path: str) -> None:
         print(f'Error reading PowerPoint file: {e}')
 
 
+def apply_git_patch(patch_str, directory='/workspace'):
```

Comment:
To maintainers: I think it is probably time for us to try to break `agentskills.py` into a folder of different submodules lol. This can belong to `agentskills/agentless.py`.

Reply:
Agree, should seperate the agentless skills into another file.